server/src/test/scala/za/co/absa/atum/server/api/http/CreateCheckpointEndpointUnitTests.scala

```diff
@@ -62,7 +62,7 @@ object CreateCheckpointEndpointUnitTests extends ZIOSpecDefault with Endpoints w
     suite("CreateCheckpointEndpointSuite")(
       test("Returns expected CheckpointDTO") {
         val response = request
-          .body(checkpointDTO1)
+          .body(checkpointDTO1.asJsonString)
```

Comment:
these conversions are not needed, it can be passed as not serialized ... the import sttp.client3.circe._ brings `circeBodySerializer` method into scope which generate BodySerializer given an implicit Encoder in scope for a given type ...